src/getDocumentCursorStream.js

```diff
@@ -72,7 +72,13 @@ function startStream(options, nextCursor) {
       : `/media-libraries/${options.mediaLibraryId}/export`,
   )
 
-  const url = `${baseUrl}?nextCursor=${encodeURIComponent(nextCursor)}`
+  const url = new URL(baseUrl)
+  url.searchParams.set('nextCursor', nextCursor)
+
+  // Type filtering is only supported for datasets.
+  if (options.types && options.types.length > 0 && options.dataset) {
+    url.searchParams.set('types', options.types.join(','))
```

Comment:
No need to specify delimiter
```suggestion
    url.searchParams.set('types', options.types.join())
```